registry/src/bin/database.rs

```diff
@@ -0,0 +1,4 @@
+//use buffrs_registry::*;
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() {}
```

Comment:
Why do we have a bin "database"?

Reply:
Honestly, I don't know what I was doing there :D I probably used that file for some quick tests and forgot to delete it.